source/jormungandr/jormungandr/scenarios/ridesharing/ridesharing_service_manager.py

```diff
@@ -160,13 +158,13 @@ def get_all_ridesharing_services(self):
         return self._ridesharing_services_legacy + list(self._ridesharing_services.values())
 
     def decorate_journeys_with_ridesharing_offers(self, response, request, instance):
-        # TODO: disable same journey schedule link for ridesharing journey?
-        if self.greenlet_pool_actived:
+        greenlet_pool_actived = instance.greenlet_pool_for_ridesharing_services
```

Comment:
'greenlet_pool_actived' can be replaced by instance.greenlet_pool_for_ridesharing_services Non?

Reply:
this is for calling the db one time